addons/binding/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/json/OpenUVJsonResult.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.openuv.json;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+import org.eclipse.smarthome.core.library.types.DateTimeType;
+import org.eclipse.smarthome.core.library.types.DecimalType;
+
+/**
+ * The {@link OpenUVJsonResult} is responsible for storing
+ * the "result" node from the OpenUV JSON response
+ *
+ * @author Gaël L'hopital - Initial contribution
+ */
+public class OpenUVJsonResult {
+
+    private DecimalType uv;
+    private ZonedDateTime uv_time;
+    private DecimalType uv_max;
+    private ZonedDateTime uv_max_time;
+    private DecimalType ozone;
+    private ZonedDateTime ozone_time;
+    private OpenUVSafeExposureTime safe_exposure_time;
```

Comment:
When the Gson instance is created with `.setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES)` you can use camel cased field names so they comply with variable naming conventions. It's also used that way in the Nest Binding.